<launch>
  <node pkg="purepursuit" exec="purepursuit_node" output="screen" >

    <!-- sub topics -->
    <remap from="/odom" to="/pf/pose/odom" />
    <remap from="/scan" to="/scan" />
    <remap from="/path_to_follow" to="/path_to_follow" />
    <remap from="/local_path" to="/local_path" />

    <!-- pub topics -->
    <remap from="/drive_parameters" to="/pp_test/drive_parameters" />
    <remap from="/pp/act_pos" to="/pp_test/spline_pos" />
    <remap from="/pp/goal" to="/pp_test/goal" />

    <!-- params file -->
    <param name="params_file" value="$(find-pkg-share purepursuit)/conf/params_phil.yaml" />
    <param name="fixed_frame" value="map" />
  </node>

  <include file="$(find-pkg-share path_publisher)/launch/path_publisher_phil.launch.xml"/>
</launch>